deploy/docs/Installation_with_Helm.md

```diff
@@ -3,18 +3,22 @@
 Our Helm chart deploys Kubernetes resources for collecting Kubernetes logs, metrics, and events;
 enriching them with deployment, pod, and service level metadata; and sends them to Sumo Logic.
 
-- [Requirements](#requirements)
-- [Prerequisite](#prerequisite)
-- [Installation Steps](#installation-steps)
-  - [Authenticating with container registry](#authenticating-with-container-registry)
-  - [Installing the helm chart in Openshift platform](#installing-the-helm-chart-in-openshift-platform)
-- [Viewing Data In Sumo Logic](#viewing-data-in-sumo-logic)
-- [Troubleshooting Installation](#troubleshooting-installation)
-  - [Error: timed out waiting for the condition](#error-timed-out-waiting-for-the-condition)
-  - [Error: collector with name 'sumologic' does not exist](#error-collector-with-name-sumologic-does-not-exist)
-- [Customizing Installation](#customizing-installation)
-- [Upgrading Sumo Logic Collection](#upgrading-sumo-logic-collection)
-- [Uninstalling Sumo Logic Collection](#uninstalling-sumo-logic-collection)
+- [Installation with Helm](#installation-with-helm)
+  - [Requirements](#requirements)
+    - [Kubernetes version](#kubernetes-version)
+  - [Prerequisite](#prerequisite)
+  - [Installation Steps](#installation-steps)
+    - [Authenticating with container registry](#authenticating-with-container-registry)
+    - [Installing the helm chart in Openshift platform](#installing-the-helm-chart-in-openshift-platform)
+  - [Viewing Data In Sumo Logic](#viewing-data-in-sumo-logic)
+  - [Troubleshooting Installation](#troubleshooting-installation)
+    - [Error: timed out waiting for the condition](#error-timed-out-waiting-for-the-condition)
+    - [Error: collector with name 'sumologic' does not exist](#error-collector-with-name-sumologic-does-not-exist)
+  - [Customizing Installation](#customizing-installation)
+  - [Upgrading Sumo Logic Collection](#upgrading-sumo-logic-collection)
+  - [Uninstalling Sumo Logic Collection](#uninstalling-sumo-logic-collection)
+    - [Post installation cleanup](#post-installation-cleanup)
+    - [Removing the kubelet Service](#removing-the-kubelet-service)
```

Comment:
Let's not. If you'd like to use the same settings in an editor you might want to add the following to e.g. vs code settings:

```
    "markdown.extension.toc.levels": "2..4",
```

Reply:
Sorry about that, apparently an extension I was using comes with an "autoupdate TOC on save" feature turned on by default. Hadn't even noticed I committed this change.

Reply:
It's fine. I also use it. Just make sure to use the above config and we're good. Perhaps we should put this somewhere for everyone to (re)use 🤔